installation-bundle/src/Database/Installer.php

```diff
@@ -238,11 +238,13 @@ private function checkEngineAndCollation(array &$sql, array &$order, Schema $fro
             $this->setLegacyOptions($table);
 
             $tableOptions = $this->connection
-                ->query("SHOW TABLE STATUS LIKE '".$tableName."'")
+                ->executeQuery(
+                    'SHOW TABLE STATUS WHERE Name = ? AND Engine IS NOT NULL AND Create_options IS NOT NULL AND Collation IS NOT NULL',
```

Comment:
```suggestion
                    'SHOW TABLE STATUS WHERE Name = ? AND Engine IS NOT NULL',
```
Wouldn’t it be enough to check if `Engine` is set?

Reply:
In theory, yes. We're accessing the other parameters as well, though and I'm not 100% sure if there are cases where some are set.

Reply:
If we want to check all features we are using should we then query the `Row_format` column too?

Reply:
Where are we using this?

Reply:
Line 255/257 `ROW_FORMAT = DYNAMIC`

Reply:
As we're only reading `Create_options` I think this is fine.

Reply:
I think I finally understand what this code does 🙈
Sorry for the confusion ☺️

Reply:
No worries at all. :slightly_smiling_face: